### PR TITLE
added rum to header search

### PR DESCRIPTION
**Pull request by shsteimer**
Test URLs:
- Before: https://main--uhg-surest--hlxsites.hlx.live/
- After: https://rum-search--uhg-surest--hlxsites.hlx.live/

### CODE REVIEW

**Comment by shsteimer** [comment]
@trieloff @davidnuescheler 

as we do final prep for surest go-live, one of their requests was that we be able to give them data on what users are searching for. I got the code for the sampleRum call here from adobe blog, so assuming this will work, but looking for your input on what (if anything) else might be needed to be able to get them that data?

File: blocks/header/header.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
-import { getMetadata, decorateIcons } from '../../scripts/lib-franklin.js';
+import { getMetadata, decorateIcons, sampleRUM } from '../../scripts/lib-franklin.js';
 import { createElement, createOptimizedPicture, decorateLinks } from '../../scripts/scripts.js';
 
 // media query match that indicates mobile/tablet width
 const isDesktop = window.matchMedia('(min-width: 900px)');
 
+let tId;
+function debounce(method, delay) {
```
Comment (badvision) [inline blocks/header/header.js:8]: Good thinking!

**Review by badvision — APPROVED** [review]
Works for me!

File: blocks/header/header.js [hunk 2]
```diff
@@ -16,6 +24,7 @@ async function execSearch(query, resultsContainer) {
 
   resultsContainer.innerHTML = '';
   if (query.length >= 3) {
+    sampleRUM('search', { source: '.search-input', target: query });
```
Comment (trieloff) [inline blocks/header/header.js:27]: Good enough. If you wanted to change your `exectSearch` function signature, you could pass the element and then use `sampleRUM.sourceselector(element)` to make it more generic, but it's probably overkill for this project.

**Review by trieloff — APPROVED** [review]

**Pull request merged**